azext_iot/central/_help.py

```diff
@@ -31,6 +31,18 @@ def load_central_help():
         short-summary: Manage IoT Central applications.
         long-summary: Create, delete, view, and update your IoT Central apps.
         """
+    helps[
+        "iot central query"
+    ] = """
+        type: command
+        short-summary: Query device telemetry or property data with IoT Central Query Language.
```

Comment:
Is IoT Central Query Language different from IoT Hub query language? Would be nice to provide a reference link to users who want to learn more.

Reply:
Added the query language link here.